python-sdk/tests/benchmark/infrastructure/kubernetes/job.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: benchmark
-        image: gcr.io/astronomer-dag-authoring/benchmark
+        image: gcr.io/astronomer-dag-authoring/benchmark:%VERSION%
```

Comment:
can you not access the env var here?

Reply:
@feluelle As per my understanding. We can't. But do let me know if you think there is a way